src/run.hh

```diff
@@ -0,0 +1,22 @@
+#ifndef RUN_H
+#define RUN_H
+
+#include "rule.hh"
+
+#ifdef __cplusplus
```

Comment:
what if it's not defined?

Reply:
This was left over from when I was compiling main.cc as C. I'll remove it.